acl_loader/main.py

```diff
@@ -579,10 +579,13 @@ def show_rule(self, table_name, rule_id):
         :param rule_id: Optional. ACL rule name. Filter rule by specified rule name.
         :return:
         """
-        header = ("Table", "Rule", "Priority", "Action", "Match")
+        header = ("Table", "Rule", "Priority", "Action", "Match", "Packets", "Bytes")
 
         ignore_list = ["PRIORITY", "PACKET_ACTION", "MIRROR_ACTION"]
 
+        counter_db = SonicV2Connector(host="127.0.0.1")
+        counter_db.connect(counter_db.COUNTERS_DB)
+
```

Comment:
could you move these two lines to below? right above the part you add the counter querying logic

Reply:
Do you mean move those two lines above the "counter_db.get_all", right? If yes, then it will re-create a socket with Redis DB for each ACL rule. IMO, it is not a good way, as we can use just one socket to get the counters for all ACL rules, no need to create connection with Redis DB again and again.